### PR TITLE
feat: remove redis references

### PR DESCRIPTION
**Pull request by SoulPancake**

### CODE REVIEW

File: docs/about/sponsors.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 linkTitle: "Sponsors"
 weight: 4
-description: Current and former Redis sponsors
+description: Current and former Valkey sponsors
 aliases:
     - /topics/sponsors
 ---
 
-From 2015 to 2020, Salvatore Sanfilippo's work on Redis was sponsored by [Redis Ltd.](https://redis.com) Since June 2020, Redis Ltd. has sponsored the [governance of Redis](/topics/governance). Redis Ltd. also sponsors the hosting and maintenance of [redis.io](https://redis.io). 
+From 2015 to 2020, Salvatore Sanfilippo's work on Valkey was sponsored by [Valkey Ltd.](https://server.com) Since June 2020, Valkey Ltd. has sponsored the [governance of Valkey](/topics/governance). Valkey Ltd. also sponsors the hosting and maintenance of [server.io](https://server.io).
```
Comment (zuiderkwast) [inline docs/about/sponsors.md:10]: No need to update this file. We can do it in another PR.

File: docs/data-types/_index.md [hunk 2]
```diff
@@ -20,92 +20,92 @@ If you'd like to try a comprehensive tutorial for each data structure, see their
...
-With a Redis set, you can add, remove, and test for existence in O(1) time (in other words, regardless of the number of set elements).
+[Valkey sets](/docs/data-types/sets) are unordered collections of unique strings that act like the sets from your favorite programming language (for example, [Java HashSets](https://docs.oracle.com/javase/7/docs/api/java/util/HashSet.html), [Python sets](https://docs.python.org/3.10/library/stdtypes.html#set-types-set-frozenset), and so on).
+With a Valkey set, you can add, remove, and test for existence in O(1) time (in other words, regardless of the number of set elements).
 For more information, see:
 
-* [Overview of Redis sets](/docs/data-types/sets/)
-* [Redis set command reference](/commands/?group=set)
+* [Overview of Valkey sets](/docs/data-types/sets/)
+* [Valkey set command reference](/commands/?group=set)
```
Comment (zuiderkwast) [inline docs/data-types/_index.md:44]: These datastructures are mentioned in #11.

We should just write "Overview of Sets", "Sets command reference", etc.

Maybe it's better to do this before everything else. Then we can avoid replacing twice ("Redis sets" -> "Valkey sets" -> "Sets"). It's easy to search/replace like this:

    "Redis string" -> "String"
    "Redis set" -> "Set"
    "Redis list" -> "List"
    "Redis hash" -> "Hash"
    "Redis sorted set" -> "Sorted set"
    "Redis stream" -> "Stream"
    "Redis geospatial indexes" -> "Geospatial indexes"
    "Redis bitmaps" -> "Bitmaps"
    "Redis bitfields" -> "Redis bitfields"
    "Redis HyperLogLog" -> "Redis HyperLogLog"

It can actually be done in a separate PR before all other changes. I think it's mostly the files under docs/data-types/ anyway.

**Review by zuiderkwast — COMMENTED** [review]
Hi! I know this PR is still a draft, but have some suggestions.

Also, this PR is too huge. I hope what is done so far is mostly automatic and be done again, if we decide to change some plans.

I think we should try to do some of these things separately. Also note which files we decided to not change.

File: docs/data-types/_index.md (hunk 2 above)
Comment (zuiderkwast) [reply docs/data-types/_index.md:44]: I made this change in this PR: #29 

@SoulPancake I hope it doesn't cause you trouble.

File: docs/about/governance.md [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
 weight: 3
 description: >
-    Governance model for the Redis open source project
+    Governance model for the Valkey open source project
 aliases:
     - /topics/governance
 ---
 
-From 2009-2020, Salvatore Sanfilippo built, led, and maintained the Redis open source project. During this time, Redis had no formal governance structure, operating primarily as a [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life)-style project.
+From 2009-2020, Salvatore Sanfilippo built, led, and maintained the Valkey open source project. During this time, Valkey had no formal governance structure, operating primarily as a [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life)-style project.
```
[the remark this answers is not in the document]
Comment (zuiderkwast) [reply docs/about/governance.md:11]: ... so it's better to not change this page at all.

**Comment by zuiderkwast** [comment]
@SoulPancake This repo is moving fast. It's not working to do all these changes in one PR.

We need to make simple changes and merge them fast. I think we shall focus on only uppercase "Redis" to "Valkey" in pages under `docs/`. Also, to be able to repeat the changes, we need to note which regex replacemens were done and which manual replacements were done, so it can be done again if there are merge conflicts or other problems.

**Comment by SoulPancake** [comment]
Hi alright
Sorry for the delay here got a little occupied
I will address this within tomorrow

**Comment by zuiderkwast** [comment]
@SoulPancake I think I did all now of them now. I got a flow with the regexes that seemed to work well. Anyhow, it's more work to review than to do the search-replace PRs, so if you want to review these, it'd be good.

Another thing would be to do client/, modules/, libraries/ or tools/. There we have to be careful where Redis is part of the name of those external projects.

**Comment by SoulPancake** [comment]
Let me take up client/ as a part of this PR and I can review the remaining in the meantime

**Comment by zuiderkwast** [comment]
Great. Don't forget `-s` or `--signoff` for `git commit`. (See CONTRIBUTING.md what it is about.) It's possible to add it afterwards if you forgot, for example by doing `git rebase --signoff HEAD~2` to add it to the last 2 commits.

I don't mind if you open new PRs or reuse this one. Whatever is easier for you.
